<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mri_sig_Widget</class>
 <widget class="QWidget" name="mri_sig_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Signals</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="plotWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="plotLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controlWidget1" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>850</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QWidget" name="widget1" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <widget class="QLabel" name="ctrlLabel_1">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>180</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Basic control panel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_z">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>330</y>
           <width>290</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>-92</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>50</number>
         </property>
        </widget>
        <widget class="QPushButton" name="startButton">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>110</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QLabel" name="gradoffsetLabel_x">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>240</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>X gradient offset</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="peak">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>490</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="fwhmLabel">
         <property name="geometry">
          <rect>
           <x>190</x>
           <y>490</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>FWHM</string>
         </property>
        </widget>
        <widget class="QPushButton" name="acquireButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>400</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Acquire</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="freqValue">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>60</y>
           <width>95</width>
           <height>24</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="decimals">
          <number>5</number>
         </property>
         <property name="maximum">
          <double>60.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.001000000000000</double>
         </property>
         <property name="value">
          <double>15.670000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="shimmingLabel">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>180</y>
           <width>260</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Shimming tools(mA of current)</string>
         </property>
        </widget>
        <widget class="QLabel" name="gradoffsetLabel_z">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>360</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Z gradient offset</string>
         </property>
        </widget>
        <widget class="QLabel" name="peakLabel">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>490</y>
           <width>41</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Peak</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_y">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>270</y>
           <width>290</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>45</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>50</number>
         </property>
        </widget>
        <widget class="QLabel" name="freqLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>131</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Frequency/MHz</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="gradOffset_z">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>360</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>-92</number>
         </property>
        </widget>
        <widget class="QLabel" name="gradoffsetLabel_y">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>300</y>
           <width>150</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Y gradient offset</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="gradOffset_x">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>240</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>120</number>
         </property>
        </widget>
        <widget class="QPushButton" name="zeroShimButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>620</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>zero shims</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>40</y>
           <width>71</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="checkBoxLayout"/>
        </widget>
        <widget class="QPushButton" name="startButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>720</y>
           <width>171</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Display settings</string>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider_x">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>210</y>
           <width>290</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>120</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksBelow</enum>
         </property>
         <property name="tickInterval">
          <number>50</number>
         </property>
        </widget>
        <widget class="QLineEdit" name="fwhm">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>490</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QPushButton" name="loadShimButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>580</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>load shims</string>
         </property>
        </widget>
        <widget class="QPushButton" name="stopButton">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>110</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="gradOffset_y">
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>300</y>
           <width>60</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="value">
          <number>45</number>
         </property>
        </widget>
        <widget class="QPushButton" name="saveShimButton">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>540</y>
           <width>150</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>save shims</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="gradOffset_disp_z">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>360</y>
           <width>44</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>-92</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="gradOffset_disp_y">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>300</y>
           <width>44</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>45</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="gradOffset_disp_x">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>230</x>
           <y>240</y>
           <width>44</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>120</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ControlWidget2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>850</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QWidget" name="widget2" native="true">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <widget class="QLabel" name="TRLabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>380</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>TR/ms</string>
         </property>
        </widget>
        <widget class="QLabel" name="ctrlLabel_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>221</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>Sequence control panel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="TELabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>340</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>TE/ms</string>
         </property>
        </widget>
        <widget class="QComboBox" name="seqType">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>110</y>
           <width>221</width>
           <height>25</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QLabel" name="seqTypeLabel">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Sequence type</string>
         </property>
        </widget>
        <widget class="QPushButton" name="uploadSeqButton">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>150</y>
           <width>161</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>upload sequence</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="para_TE">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>340</y>
           <width>71</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>6</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
        <widget class="QSpinBox" name="para_TR">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>380</y>
           <width>71</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>1500</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
        <widget class="QLabel" name="seqParaLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>300</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Sequence parameters</string>
         </property>
        </widget>
        <widget class="QPushButton" name="seqButton">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>460</y>
           <width>181</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="text">
          <string>confirm sequence</string>
         </property>
        </widget>
        <widget class="QLabel" name="uploadSeqLabel">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>190</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Uploaded seq:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="uploadSeq">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>220</y>
           <width>221</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>none</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="FALabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>420</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>FA/°</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="para_FA">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>420</y>
           <width>71</width>
           <height>22</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>90</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
        <widget class="QLabel" name="calibrationLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>550</y>
           <width>121</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>Calibration</string>
         </property>
        </widget>
        <widget class="QPushButton" name="seqButton_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>590</y>
           <width>221</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>find central frequency</string>
         </property>
        </widget>
        <widget class="QPushButton" name="seqButton_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>630</y>
           <width>221</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>auto shim</string>
         </property>
        </widget>
        <widget class="QPushButton" name="seqButton_4">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>670</y>
           <width>221</width>
           <height>30</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>true</strikeout>
          </font>
         </property>
         <property name="text">
          <string>flip angle calibration</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
